README.md

```diff
@@ -153,6 +153,35 @@ terraform:
   # ...
 ```
 
+Sometimes you may want not to HTML-escape Terraform command outputs.
+For example, when you use use code block to print command output, it's better to use raw characters instead of character references (e.g. `-/+` -> `-/&#43;`, `"` -> `&#34;`).
```

Comment:
😎 
```suggestion
For example, when you use code block to print command output, it's better to use raw characters instead of character references (e.g. `-/+` -> `-/&#43;`, `"` -> `&#34;`).
```

Reply:
👍